content/docs/other-guides/kustomize.md

```diff
@@ -18,102 +18,155 @@ want to include in your Kubeflow deployment.
 This section describes how kfctl works with kustomize to set up your 
 Kubeflow deployment.
 
-### The Kubeflow installation process
+### The Kubeflow deployment process
 
 Kfctl is the Kubeflow CLI that you can use to set up a Kubernetes cluster with 
 Kubeflow installed, or to deploy Kubeflow to an existing Kubernetes cluster. 
 See the [Kubeflow getting-started guide](/docs/started/getting-started/) for
 installation instructions based on your deployment scenario.
+For example, see the guide to 
+[deploying Kubeflow with kfctl_k8s_istio](/docs/started/k8s/kfctl-k8s-istio/).
 
-The deployment process consists of three steps, _init_, _generate_, and 
-_apply_, so that you can modify your configuration before actually deploying 
-Kubeflow.
+The kfctl deployment process includes the following commands:
 
-* `kfctl init` - one time set up.
-* `kfctl generate` - creates config files defining the various resources.
-* `kfctl apply` - creates or updates the resources.
+* `kfctl build` - (Optional) Creates configuration files defining the various
+  resources in your deployment but does not deploy Kubeflow.
+  You only need to run `kfctl build` if you want
+  to edit the resources before running `kfctl apply`.
+* `kfctl apply` - Creates or updates the resources.
+* `kfctl delete` - Deletes the resources.
 
 ### Specifying a configuration file when initializing your deployment
 
 When you install Kubeflow, the deployment process uses one of a few possible
-YAML configuration files to bootstrap the configuration.
-
-Typically, you specify the configuration file with a `--config` parameter
-when you run `kfctl init`. For example:
-
-```
-export KFAPP="<your choice of application directory name>"
-export CONFIG="{{% config-uri-existing-arrikto %}}"
-kfctl init ${KFAPP} --config=${CONFIG} -V
+YAML configuration files to bootstrap the configuration. As an example,
+this guide uses the 
+[kfctl_k8s_istio.yaml](https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_k8s_istio.yaml)
```

Comment:
This replaces one config with the other.
What are the reasons for replacing the existing_arrikto example here?
Maybe we can add a link to the available configs and then refer to $CONFIG in the instructions? 
Same for the other places where specific configs are mentioned.

Reply:
I swapped to the kfctl_k8s_istio config because it's a more basic config requiring less initial setup, which makes it a good example. I've adjusted the content to provide more pointers to the list of configs and to clarify that this is just an example. I've also removed the explicit example for the GCP config. PTAL.

Reply:
Saw the changes, everything looks great!